### PR TITLE
Experimental features placeholder

### PR DESCRIPTION
**Pull request by moxiegirl**
Location in support of #13023.  I haven't added it to the toc yet.  I need to see how much material we get.

Signed-off-by: Mary Anthony mary@docker.com

ping @icecrime

### CODE REVIEW

**Comment by SvenDowideit** [comment]
Can we add / reserve a place in the mkdocs.yml TOC (commented out if you like) so we know where its going?

**Comment by moxiegirl** [comment]
@SvenDowideit I was waiting to see the comments --- wasn't clear there was only going to be a single page.

**Comment by calavera** [comment]
I think it would make sense to have separated pages per experimental feature. Definitely, this is a good starting point.

**Comment by moxiegirl** [comment]
@calavera short things like command flags wouldn't make sense in a small file. For example, if we had 3 experimental features all of which added a flag to run, maybe just one note with a list of the changes to the command.  I wanted to see some examples of what we are adding --- I have yours from this morning. But large new issues, definitely agree, I expected in their own files.

I'll add a mkdocs.yml entry and send this back out for Sven to approve. I hope. :-D

**Comment by icecrime** [comment]
@moxiegirl What do you think about a standard layout for experimental feature pages? Something such as:

- Description
- How to use it
- Known issues & limitations

The "How to use" section would say "this feature introduces a new command-line flag to the `docker run` command, bla bla bla".

**Comment by SvenDowideit** [comment]
@moxiegirl LGTM with expected `mkdocs.yml` change and we'll work from there :)

**Comment by moxiegirl** [comment]
@icecrime I think that could work really well. If the feature was super detailed, we could link to the "long doc" from the **Description**. I'll mock it up in the morning.

**Comment by thaJeztah** [comment]
> What do you think about a standard layout for experimental feature pages? Something such as..

Good idea; I think a "**RFC/feedback**" section would be good there as well; this section should contain a link to an issue on the issue-tracker for discussing the feature.

**Comment by moxiegirl** [comment]
ping @icecrime

File: docs/sources/experimental/experimental.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+page_title: Overview of Experimental Features
+page_keywords: experimental, Docker, feature
+
+# Overview 
+
+This page contains a list of features in the Docker engine which are experimental as of the current release. Experimental features are **not** ready for production.  They are provided to for test and evaluation in your sandbox environments.
```
Comment (tiborvass) [inline docs/sources/experimental/experimental.md]: `to for` is correct?

**Comment by moxiegirl** [comment]
image

This is what it looks like

File: docs/sources/experimental/experimental.md (hunk 1 above)
Comment (moxiegirl) [inline docs/sources/experimental/experimental.md]: nope. visual discrepancy or VD as we call it in the writing community.

File: docs/sources/experimental/experimental.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+page_title: Overview of Experimental Features
+page_keywords: experimental, Docker, feature
+
+# Overview
```
Comment (moxiegirl) [inline docs/sources/experimental/experimental.md]: This should say Experimental Features

**Comment by thaJeztah** [comment]
Yup, I like this as a template.

Should the introduction explain how to obtain and/or build Docker with the "experimental" features enabled? (it's not a "flag" that can be enabled at runtime)

**Comment by moxiegirl** [comment]
@icecrime Are we going to offer an experimental binary or must they build the project special? If they have to build, well, building a product like this is a pretty advanced task.  Maybe we link to that how-to  information in the repo --- with the code they are building;

**Comment by duglin** [comment]
I would be great if there was a page like: https://master.dockerproject.com/  that offered the experimental version of the builds. or a sub-dir from there.

**Comment by icecrime** [comment]
@moxiegirl @duglin We're thinking having something like https://experimental.docker.com serving an install script just like https://get.docker.com does

**Comment by moxiegirl** [comment]
@icecrime coolio.  I can add install instructions at the top as per @thaJeztah's suggestion.

**Comment by thaJeztah** [comment]
In case you need to/want to refer to *building* an experimental build, there's a `DOCKER_EXPERIMENTAL` environment variable to do so, see https://github.com/docker/docker/pull/13338.

Experimental builds can be "recognized" by the `ExperimentalBuild` property (true/false), visible in `docker info`. In addition, a "Running experimental build" warning is output in the logs

**Comment by SvenDowideit** [comment]
@moxiegirl can you re-flow to 80 char lines please - it makes diffs so much easier to parse.

**Comment by moxiegirl** [comment]
per @icecrime I"m going to merge this now and do all the fixes after.

**Pull request merged**